src/closestTo/test.ts

```diff
@@ -0,0 +1,110 @@
+// @flow
+/* eslint-env mocha */
+
+import assert from 'assert'
+import closestTo from '.'
+
+describe('closestTo', function() {
+  it('returns the date from the given array closest to the given date', function() {
+    const date = new Date(2014, 6 /* Jul */, 2)
+    const result = closestTo(date, [
+      new Date(2015, 7 /* Aug */, 31),
```

Comment:
WDUT about using the initialization by formatted string date? 

eg.

```suggestion
      new Date("2015-08-31 00:00:00"),
```

Reply:
That is a good idea!

But I'm trying to keep these consistent with the way they're used in the entire library tests, at least in this first round of migrations. 

Also, I think the way it is, leads to a more readable, more maintainable, and easier to contribute code. 

Do you agree with my point? :D

Reply:
Let's keep it as is. Besides `"2015-08-31 00:00:00"` is an invalid ISO string.